app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :users
```

Comment:
- [ ] The same here 🤔 The Comments belongs-to user through the `'author_id'` column of the Comment table. Now, we need to show that, the  `'author_id'` reference  User class modal.
Your code should look like this 👇🏻 :

```ruby
  # app/models/comment.rb
  belongs_to :author, class_name: 'User'
```